xml/containers-deploy-run.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?asciidoc-toc?><?asciidoc-numbered?><chapter xmlns="http://docbook.org/ns/docbook" xmlns:xl="http://www.w3.org/1999/xlink" version="5.0" xml:lang="en">
+<info>
+<title>Deploying and Running Containers</title>
+<date>2020-07-27</date>
```

Comment:
Please remove this date -- when this section is enabled and we forget about the existence of this date, people will be confused why the new guide ended up with an old section.